draft-ietf-rats-corim.md

```diff
@@ -1121,18 +1122,22 @@ The Reference Values Triple has the following structure:
 
 The `reference-triple-record` has the following parameters:
 
-* `ref-env`: Search criterion that locates an Evidence environment that matches the reference environment.
-* `ref-claims`: Search criteria that locates the Evidence measurements that match the reference Claims.
+* `ref-env`: Reference Environment Identity of the Target Environment
+
+* `ref-claims`: One or more measurement claims for the Target Environment
 
 To process `reference-triple-record` both the `ref-env` and `ref-claims` criteria are compared with Evidence entries.
+First `ref-env` is used as a Search criterion to locate the Evidence environment that matches the reference environment.
```

Comment:
```suggestion
First `ref-env` is used as a search criterion to locate the Evidence environment that matches the reference environment.
```

Reply:
However, this seems phrased backwards, isn't it?  One uses the environment identifiers associated with the appraised evidence to locate the matching ref-triples.